buyer-api/src/facades/getOrdersFacade.js

```diff
@@ -72,7 +65,7 @@ const getDataOrderDetails = async (order) => {
   ]);
 
   return {
-    orderAddress: order.address,
+    address: order.address,
```

Comment:
This change impacts absolutely everywhere. Let's better keep it as `orderAddress`